docs/notebooks/text_to_image.ipynb

```diff
@@ -280,7 +281,7 @@
       "cell_type": "code",
       "source": [
         "text_da = DocumentArray([Document(text='some text to encode')])\n",
-        "image_da = DocumentArray([Document(uri='my-image.png')])\n",
+        "image_da = DocumentArray([Document(uri='https://www.collinsdictionary.com/images/full/apple_158989157.jpg')])\n",
```

Comment:
Not a free-use image.  Try to use free-use images with known origins and stable URLs. Like this one: https://upload.wikimedia.org/wikipedia/commons/4/4e/Single_apple.png